draft/2022-03-16-this-week-in-rust.md

```diff
@@ -199,6 +199,10 @@ Email the [Rust Community Team][community] for access.
 
 # Rust Jobs
 
+**Juspay**
+
+* [Juspay Router - building financial switch using Rust](https://www.upwork.com/jobs/~0106e33943207a0fc6)
```

Comment:
Hello there! We format our job postings as:

Job Title (Location)

Would you be willing to format this posting as 

```
* [Freelance Rust Developer (Remote)](https://www.upwork.com/jobs/~0106e33943207a0fc6)
```

Reply:
Done. Updated

Reply:
Perfect! Thank you!